apps/projects/contracts/Projects.sol

```diff
@@ -121,12 +135,21 @@ contract Projects is AragonApp {
     event FulfillmentAccepted(bytes32 repoId, uint256 issueNumber, uint fulfillmentId);
     // Fired when settings are changed
     event BountySettingsChanged();
+    // Fired when user requests issue assignment
+    event AssignmentRequested(address applicant, bytes32 indexed repoId, uint256 issueNumber);
```

Comment:
I don't know if we need to fire the address for these events, we'll have to re-sync all of the info for the issues regardless. Thoughts?

Reply:
Yeah I agree. I'll remove it